public/data/7flash.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "Berlian Gur",
+  "type": "personal",
+  "bio": "Software Engineer",
+  "avatar": "https://avatars.githubusercontent.com/u/4569866?v=4",
```

Comment:
It is best to use this link as it's shorter and updates automatically.
```suggestion
  "avatar": "https://github.com/7flash.png",
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient